### ISSUE TITLE
Icon looks distorted

### ISSUE THREAD

**Issue by hrzafer** [dustinkredmond/FXTrayIcon#32]
My icon is 16x16. `SystemTray.getSystemTray().getTrayIconSize()` returns `java.awt.Dimension[width=16,height=16]` as well. But sometimes the icon looks distorted as in the image:

![image](https://user-images.githubusercontent.com/4048197/145696830-7c80793a-8770-4905-a7d3-4fdc4a28d9bf.png)

How can I fix that? Have you guys faces a similar issue?

**Comment by EasyG0ing1** [comment]
@hrzafer If you're using a mac, then use an icon that is at least 24 x 24 pixels and define it as such in FXTrayIcon. I'v discovered that with the Mac, is doesn't really matter what dimensions you give FXTI above something like 22 x22 or somewhere around there if I remember correctly, and it ends up looking very nice... This key, for example, is being presented by FXTrayIcon and as you can see ... there are even noticeable details in the cut portion of the key...


<img width="90" alt="Screen Shot 2021-12-12 at 12 39 12 AM" src="https://user-images.githubusercontent.com/10106834/145705927-06c5652a-5991-43cd-aa1a-a5221ae366f9.png">
.

**Comment by hrzafer** [comment]
The image above is from windows-11. This one is from windows-10: ![image](https://user-images.githubusercontent.com/4048197/145901384-1a29c252-6f61-468a-aaad-cc1613859c19.png)

icon looks OK on the system tray:

![image](https://user-images.githubusercontent.com/4048197/145901533-d9fe1fc7-5edb-4d48-bd9c-8760779e3d6a.png)

**Comment by EasyG0ing1** [comment]
@hrzafer - I was just looking at the Oracle sample project for the AWT TrayIcon and they posted a gif file that they used for their icon. I copied that file and put a star in it. Here is how they create the image for the TrayIcon object:

`final TrayIcon trayIcon = new TrayIcon(createImage("images/bulb.gif", "tray icon"));`

USing this method to actually create the image:

```
 protected static Image createImage(String path, String description) {
    URL imageURL = TrayIconDemo.class.getResource(path);
       
    if (imageURL == null) {
        System.err.println("Resource not found: " + path);
        return null;
    } else {
        return (new ImageIcon(imageURL, description)).getImage();
    }
}
```

And here is the gif that I made from theirs:

![star](https://user-images.githubusercontent.com/10106834/145988316-491a95d8-cb99-4773-891a-0826c752310e.gif)


Not sure it will be of any use to you, but it might be worth trying it their way to see if you get any different results.
